### PR TITLE
Remove sorting pubs option on deployed site

### PR DESCRIPTION
**Pull request by pmyip3**
# Description 
- Remove sorting publication option on deployed site
- Remove the pagination if the items can fit within a page
- Refactor the sorting publications function to make it stable

# Type of change 
- [ ] Bug fix
- [ ] New feature
- [ ] Refactoring
- [ ] Documentation 
- [x] Improvement

# Problem
- Sorting publications options not required on deployed site 
- The current sorting function is not stable --> same options might lead to slightly different sorting order  

# Solution description
See description 

# Tests 
All unit tests pass? N/A

Builds successfully? Yes

Attach screenshot of the expected result (e.g. UI of the website/ postman api call result/ database changes)
- sorting/layout options removed, pagination removed if total publications less than 10 (can be fit within a page)
image

# Relevant document/ link (if any) N/A

# Risk
Low. Simple change on UI

### CODE REVIEW

File: scholly/base/src/config/publications.js
Comment (pmyip3) [inline scholly/base/src/config/publications.js:21]: Need to remove this after #194 merged to main

**Comment by vgoh1** [comment]
Should you also remove the pagination on the publications page on the main site if there's less than 10 publications?
image

**Review by vgoh1 — APPROVED** [review]

File: scholly/base/src/components/publications/Publications.js
Comment (aksh0001) [inline scholly/base/src/components/publications/Publications.js:16]: if we're removing sorting in the `base`, shouldn't this functionality be removed too?

Comment (pmyip3) [reply scholly/base/src/components/publications/Publications.js:16]: As the publications in the db are't sorted, this function is still needed for sorting the publication in the first rendering based on the layout and sorting preference stored in the db

Comment (aksh0001) [reply scholly/base/src/components/publications/Publications.js:16]: SGTM, relsving.

File: scholly/base/src/components/publications/Publications.js
Comment (aksh0001) [inline scholly/base/src/components/publications/Publications.js:4]: @vgoh1 we should keep an eye out if the `Fragment` import throws a check when changes from main are merged with the updated SA. I expect it should.

**Review by aksh0001 — APPROVED** [review]
LGTM, could you merge this PR after the PR with the updated SA is merged?

**Review by cyr17 — APPROVED** [review]

**Pull request merged**
